.travis.yml

```diff
@@ -1,13 +0,0 @@
----
-
-language: java
-jdk:
-  - openjdk8
-before_install:
-  - cp settings*.xml ~/.m2/
```

Comment:
I noticed this (or these?) wasn't (weren't) in the GitHub Action so went to look at the file. The default one looks like it's empty?

Is it actually used or could it be removed from this repo? I assume the security version is used somewhere, but is not needed in the GitHub Action (because Travis was deploying too or something)? What's the story with these?

Reply:
settings.xml looks generic... don't remember why it's here (unless tied to the security,xml).
The security file was needed because, when app was first Mavenized, Oracle JDBC jars weren't available in standard Maven repos and had to be retrieved from Oracle, which required a username/password. I'll experiment and verify these can be removed.